hostinfo/packages.lua

```diff
@@ -17,7 +17,7 @@ local HostInfo = require('./base').HostInfo
 
 local fmt = require('string').format
 local los = require('los')
-local sctx = require('sigar').ctx
+local vendor = require('sigar'):new():sysinfo().vendor:lower()
```

Comment:
let's change this to just ```require('sigar')```, then move the vendor to where we use it... that way we aren't holding a global variable (the module itself is cached).